themes/src/main/resources/theme/base/admin/resources/partials/client-service-account-roles.html

```diff
@@ -7,10 +7,23 @@
 
     <kc-tabs-client></kc-tabs-client>
 
-    <h2><span>{{client.clientId}}</span> {{:: 'service-accounts' | translate}} </h2>
-    <p class="subtitle"></p>
-
     <form class="form-horizontal" name="realmForm" novalidate kc-read-only="!client.access.configure" data-ng-show="client.serviceAccountsEnabled">
+
+        <h2>{{:: 'service-account' | translate}}</h2>
+
+        <div class="form-group">
+            <label class="col-md-2 control-label">{{:: 'user-name' | translate}}</label>
```

Comment:
Would it be better with "Service Account User" instead of username?

Or just a link "To manage details and group mappings go here"?

Reply:
I adopted your suggestion and Renamed the field to "Service Account User". The tooltip now says: "Username the Service Account. To manage details and group mappings click on the username."